### ISSUE TITLE
JS errors

### ISSUE THREAD

**Issue by nmoreaud** [algenty/grafana-flowcharting#88]
I installed your plugin in the latest docker grafana image (v6.5.0) with zabbix plugin to query data.
When I add a FlowCharting panel, I've got the following JS errors in firefox console panel :
```
Plugin component error TypeError: "$rootScope is undefined"
    FlowchartCtrl flowchart_ctrl.js:99
    Angular 5
    d variableQueryEditorLoader.tsx:21
    d variableQueryEditorLoader.tsx:26
    link ChangeTracker.ts:19
ChangeTracker.ts:23:12

TypeError: "this.flowchartHandler is undefined"
TypeError: "this.flowchartHandler is undefined"
...
```

I cannot create charts.
Thank you in advance.

![Screenshot](https://user-images.githubusercontent.com/2413781/69645212-16982600-1066-11ea-9087-ff0561e1be0a.png)

**Comment by algenty** [comment]
Yes, i have notice it to grafana
https://github.com/grafana/grafana/issues/20255

You can use 0.6.0 in download section for this problem (not published à this time)
https://algenty.github.io/flowcharting-repository/ARCHIVES.html

Give me some news if you use it

BR
Arnaud

**Comment by nmoreaud** [comment]
You're right, this is fixed in 0.6 !
Thanks a lot
